plugins/plugin.h

```diff
@@ -46,11 +46,23 @@ struct qemu_plugin_state {
     struct qht dyn_cb_arr_ht;
 };
 
+typedef void (*cb_func_t) (gpointer evdata, gpointer udata);
+
+struct qemu_plugin_qpp_cb {
+    const char *plugin;
+    const char *name;
+    cb_func_t registered_cb_funcs[32];
```

Comment:
Should we use a typedef here, perhaps just using QEMU_PLUGIN_EV_MAX again? Or make a new one for QEMU_PLUGIN_QPP_MAX?

Reply:
I think reusing QEMU_PLUGIN_EV_MAX works